modular_skyrat/modules/barricades/code/barricade.dm

```diff
@@ -690,7 +690,7 @@
 	if(.)
 		return
 
-	if(do_after(user, 50, src))
+	if(do_after(user, 20, src))
```

Comment:
Could be interesting to make this time be a variable of the barricade itself, so it could be modified both by admins and in general by subtypes (for traitor items or special loot, I guess. Not that it's incredibly powerful on its own, but...)
```suggestion
	if(do_after(user, 2 SECONDS, src))
```

Reply:
I was planning to do this myself, but due to some IRL stuff couldn't. Will do, because I do agree with this idea.